cluster/common.sh

```diff
@@ -322,15 +328,18 @@ function find-tar() {
 #   SERVER_BINARY_TAR
 #   KUBE_MANIFESTS_TAR
 function find-release-tars() {
+  # shellcheck disable=SC2034
   SERVER_BINARY_TAR=$(find-tar kubernetes-server-linux-amd64.tar.gz)
   if [[ "${NUM_WINDOWS_NODES}" -gt "0" ]]; then
+    # shellcheck disable=SC2034
     NODE_BINARY_TAR=$(find-tar kubernetes-node-windows-amd64.tar.gz)
   fi
 
   # This tarball is used by GCI, Ubuntu Trusty, and Container Linux.
   KUBE_MANIFESTS_TAR=
   if [[ "${MASTER_OS_DISTRIBUTION:-}" == "trusty" || "${MASTER_OS_DISTRIBUTION:-}" == "gci" || "${MASTER_OS_DISTRIBUTION:-}" == "ubuntu" ]] || \
      [[ "${NODE_OS_DISTRIBUTION:-}" == "trusty" || "${NODE_OS_DISTRIBUTION:-}" == "gci" || "${NODE_OS_DISTRIBUTION:-}" == "ubuntu" || "${NODE_OS_DISTRIBUTION:-}" == "custom" ]] ; then
+    # shellcheck disable=SC2034
```

Comment:
why?

Reply:
This function's return value is used in if statements as below:
```bash
function verify-release-tars() {
  if ! find-release-tars; then
    download-release-binaries
  fi
}
```
I can not just export these variable in the next line, or the function will always return 0.
Do you have better solution?

Reply:
update comment

Reply:
huh, if you export on the next line it will fail on this line if it fails and not get to the export line?

Reply:
I modified this function `find-release-tars`, split the vars and it return value, it is a little verbose, please check again.

Reply:
still disabling?